azure-pipelines.yml

```diff
@@ -98,3 +98,40 @@ jobs:
       configuration: 'Release'
       msbuildArgs: '/p:AppxPackageSigningEnabled=false'
     displayName: 'Build UWP'
+    
+- job: androidJNI
+  pool:
+    vmImage: 'macOS-10.14'
```

Comment:
is this right?

Reply:
Yes! More tools are available on the mac image for Android. It can also works on Windows vm but Gb of tools have to be unzipped and installed.